data/list/basic.lean

```diff
@@ -3880,9 +3900,10 @@ end nodup
 section erase_dup
 variable [decidable_eq α]
 
-/-- `erase_dup l` removes duplicates from `l` (taking only the first occurrence).
+/-- `erase_dup l` removes duplicates from `l` (leaving only the last occurrence).
+  Defined as `pw_filter (≠)`.
 
-     erase_dup [1, 2, 2, 0, 1] = [1, 2, 0] -/
+     erase_dup [1, 2, 2, 0, 1] = [2, 0, 1] -/
```

Comment:
hm, that example was meant to show how the order is not maintained but the order of the selected elements is. With the new example it looks like a right sequence will suffice. Maybe just flip the whole thing?
```suggestion
     erase_dup [1, 0, 2, 2, 1] = [0, 2, 1] -/
```